lib/signaling/v2/networkqualitysignaling.js

```diff
@@ -96,6 +99,13 @@ class NetworkQualitySignaling extends MediaSignaling {
       _networkQualityInputs: {
         value: new AsyncVar()
       },
+      _resendTimer: {
+        value: new Timeout(() => {
+          // and schedule next timer for double the delay..
+          this._resendTimer.setDelay(this._resendTimer.delay * 2);
```

Comment:
This seems too steep of a curve. Thoughts on using a lower multiplier? 1.3 maybe?

Reply:
Okay, I have now proposed that we do it at multiplier of 1.5 (https://code.hq.twilio.com/client/room-signaling-protocol/pull/78) Once that PR is approved I will code with whatever we agree.